apps/frontend/src/app/Components/NoArtWarning.tsx

```diff
@@ -0,0 +1,15 @@
+import { Alert, Link } from '@mui/material'
+import { Link as RouterLink } from 'react-router-dom'
+
+export default function NoArtWarning() {
+  return (
+    <Alert severity="warning" variant="filled">
```

Comment:
We have a translation key for this one, we should add it to this shared component